src/Domain/Product.php

```diff
@@ -0,0 +1,105 @@
+<?php declare(strict_types = 1);
+
+namespace SandwaveIo\RealtimeRegister\Domain;
+
+use SandwaveIo\RealtimeRegister\Domain\Enum\CertificateTypeEnum;
+use SandwaveIo\RealtimeRegister\Domain\Enum\FeatureEnum;
+use SandwaveIo\RealtimeRegister\Domain\Enum\ValidationTypeEnum;
+
+final class Product implements DomainObjectInterface
+{
+    public string $product;
+    public string $brand;
+    public string $name;
+    public string $validationType;
+    public string $certificateType;
+    /** @var string[] */
+    public array $features;
+    /** @var int[] */
+    public array $periods;
+    public int $warranty;
+    public string $issueTime;
+    public int $renewalWindow;
+    public int $includedDomains;
+    public int $maxDomains;
+    /** @var string[] */
+    public array $requiredFields;
+    /** @var string[] */
+    public array $optionalFields;
+
+    private function __construct(
```

Comment:
https://dm.realtimeregister.com/docs/api/ssl/products/get

features, requiredFields, optionalFields, includedDomains and maxDomains are all optional, shouldn't these be nullable?